src/interface/ttk-thirdparty-api.yaml

```
@@ -2230,6 +2230,110 @@ paths:
           $ref: '#/paths/~1otp/post/responses/501'
         '503':
           $ref: '#/paths/~1otp/post/responses/503'
+  /store/consentRequests:
```

```suggestion
/store/consentRequest/{ID} 
```
when we will use a generic storage handler from TTK we don't have to implement the logic of how to extract the ID from the payload.

---

ok